measuring/gqm_example/gqm_example_metric.md

```diff
@@ -0,0 +1,23 @@
+# Example Metric
+
+**A short name of the metric**
+
+A brief description of the metric.
+
+**Synopsis**: A brief summary of the metric.  
+
+**Scale/UoM**: What scale and unit of measure does the metric use? The scale determines what kinds of operations are possible. For example, you cannot calculate averages on a nominal scale, even if the ordinals are expressed as numbers. The scale types are nominal, ordinal, interval, and ratio.  
```

Comment:
What does this mean?  We need to explain it simpler.

Reply:
I tried making @gruetter 's guidance easier to understand, but maybe it's more complicated?

Here's the original content.

**Scale/UoM**: What is the scale and unit of measure of the metric? (nominal, ordinal, interval, ratio - the scale limits the types of operations you can do. E. g. you can't compute averages on a nominal scale, even if the ordinals are expressed as numbers)

Maybe we need a diagram?

```mermaid
graph TD;
A[Scale/UoM] --> B[What is the scale and unit of measure of the metric?];
B --> C1[nominal];
B --> C2[ordinal];
B --> C3[interval];
B --> C4[ratio];
C1 --> D1[Can't compute averages even if expressed as numbers];
```

Maybe we need something like this?

![image](https://github.com/InnerSourceCommons/managing-inner-source-projects/assets/776901/b69acb46-0d84-4416-88a0-d23cdc669f78)

Reply:
Oh I see it was some text from @gruetter.  I went and looked it up, but not sure what point of confusion are we trying to solve by including this section?  Thanks for helping me to understand.

Reply:
I haven't encountered confusion around this, but I presume @gruetter has.

I wonder if there's a middle-ground where we define the metric data type and let people consider details about the unit of measure themselves?